### DISCUSSION TITLE
Never able to pass the Github student verification

### DISCUSSION THREAD

**Discussion by KMint1819** [community/community#51292] [GitHub Education] [answered]
Hello. I've submitted my academic affiliation for more than 5 times, with different angles of different documents (admission letter, student ID...) and filled up every blank in my personal information but I never managed to pass the verification. Any ideas? Thanks.

**Comment by eefernet** [comment]
Same. There signup process is abhorrent. I have tried on my mac and used even my iphone as a camera. Cannot seem to get a good enough photo. Desktop is also broken. I think i glitched it to upload a direct photo but idk how it worked to do it again.

**Comment by KMint1819** [comment, marked as answer]
Just passed the verification after 10+ attempts. Setting the primary email to the school email works for me.

**Reply by childerx** [reply]
I have used my school's email and everything regarding the school; but my application is still being rejected. I have tried almost 10+ times now

**Reply by KrisITB** [reply]
Lucky you, but I would appreciate it if you could mark it as unsolved as it doesn't work for me and I bet it doesn't work for many

**Comment by GustavoHASH** [comment]
my college does not provide institutional email to students, github education was impossible to obtain, congratulations to those involved

**Comment by CodecNomad** [comment]
We need to somehow contact human support, but that's close to impossible. Also, it's the precheck that's messing you guys up.

**Reply by eefernet** [reply]
yes, i have been trying to find a way to reach out to them to explain my case but there are no options it seems. I ended up just paying for a year lol

**Reply by CodecNomad** [reply]
I just had to remove the school location and it was fixed, however now I need to find a way to send 2 images.

**Reply by CodecNomad** [reply]
Fixed boys!!! All I had to do was,  open up my online school portal in the background (on my monitor) put my student id's back next to it so it has date and take a screenshot from my phone! Hope you guys will be able to get it too

**Comment by garciavalter** [comment]
Got once accepted for it, but now I'm not able to renew... every try gets rejected :(

**Reply by CodecNomad** [reply]
Hi! I got it fixed above you, might try what I did.

**Comment by s0566251** [comment]
same here..

**Comment by serpsaipong-nav** [comment]
My student id is not in English. Is that why my verification is not pass? I can't find it in English. How can I pass this verification T T

**Reply by CodecNomad** [reply]
Neither was mine.

**Comment by SubMrge** [comment]
> Hello. I've submitted my academic affiliation for more than 5 times, with different angles of different documents (admission letter, student ID...) and filled up every blank in my personal information but I never managed to pass the verification. Any ideas? Thanks.

annoying thing. It says they cannot accept it after 4 attempts. No matter what i do.

**Comment by Kellyfiocchi** [comment]
Hello. I've submitted my academic affiliation for more than 6times, with different angles of different documents (admission letter, student ID...) and filled up every blank in my personal information but I never managed to pass the verification. Any ideas? Thanks.

**Comment by KrisITB** [comment]
Same problem here. Did all I was asked for...even setting my full name on my account (which sux, you have it on the scan if needed) and yet still failed validation... I've been going through this process several times in the past without any problems... what a wonderful update... stopping students from renewing their student status is such a great learning experience that could be wasted on learning new stuff instead of leveling up your skills and preparing you to dealing with bots (latte is obviously far more important)... welcome to 2023!

**Reply by KrisITB** [reply]
update: I get less errors now! 

...I made a jpeg (as for some reasons pdf or png is not an option here) scan with 600dpi for you and:

Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.

...last try will be with a phone as it looks that scanners are out of fashion these days

**Reply by KrisITB** [reply]
nope, I give up, I just wasted 2 hours of my life for this nonsense

**Comment by harshitk024** [comment]
Hi, you can raise a ticket and tell them about your issue , your problem will be solved.
you can create a ticket by this link -  [Create Ticket](https://support.github.com/contact/education)

**Reply by KrisITB** [reply]
![image](https://user-images.githubusercontent.com/11059515/229581588-0b6204df-5cf1-4717-890f-2d003d6c48c0.png)

**Reply by KrisITB** [reply]
round two on a different path and ticket closed in seconds with such a nice "get lost": 
![image](https://user-images.githubusercontent.com/11059515/229583709-3f8304d5-5d94-4c0c-9aca-4a80c1887bc9.png)

**Reply by harshitk024** [reply]
[image](https://user-images.githubusercontent.com/11059515/229583709-3f8304d5-5d94-4c0c-9aca-4a80c1887bc9.png)
> 

click  on first  link and reapply again 
If you are using same validation id,
This time use different validation proof
Like - enrollment letter with the date.
It should  be verified

**Reply by KrisITB** [reply]
did that, several times, didn't work

**Reply by harshitk024** [reply]
Looks like there is another issue in your 
Application, keep trying.

**Comment by xBLACKICEx** [comment]


**Reply by CodecNomad** [reply]
> I also feel a bit disgusted by a rule, why I give github company to provide proof that I am a student, it is necessary to disclose personal information to all people on the Internet other people have anything to do with github(publie user profile, name(real full name) age location etc...)?

Feels like they are trying to discourage you from using this pack.

**Comment by OnlineBunker** [comment]
<h3> Verification Error</h3>
<h5> I've done all i could but the website on which the verification takes place, it doesn't support my phone's back camera(only front is visible) and on my laptop it accesses my camera but no affirmation that the photo has been clicked. </h5> 
<p>After somehow scanning my School ID, it says that it should contain the same name as your account does, and the worst part is that it IS the same name</p>
